### ISSUE TITLE
Wrong form input id's and label for's for radio and checkbox input (1.1)

### ISSUE THREAD

**Issue by mikebranderhorst** [fuel/core#811]
Like the example in docs:

```
echo Form::label('Male', 'gender');
echo Form::radio('gender', 'Male', array('checked' => 'checked'));
echo Form::label('Female', 'gender');
echo Form::radio('gender', 'Female');

echo Form::label('Tennis', 'sports');
echo Form::checkbox('sports[]', 'tennis', array('checked' => 'checked'));
echo Form::label('Volleyball', 'sports');
echo Form::checkbox('sports[]', 'volleyball');
```

gives:

```
<label for="form_gender">Male</label>
<input checked="checked" name="gender" value="Male" type="radio" id="form_gender" />
<label for="form_gender">Female</label>
<input name="gender" value="Female" type="radio" id="form_gender" />

<label for="form_sports">Tennis</label>
<input checked="checked" name="sports[]" value="tennis" type="checkbox" id="form_sports[]" />
<label for="form_sports">Volleyball</label>
<input name="sports[]" value="volleyball" type="checkbox" id="form_sports[]" />
```

the for is the same in the two labels
the id is the same (and thus not unique) in the two inputs

maybe it should generate the value as well

id="form_gender_male"
id="form_gender_female"

**Comment by jschreuder** [comment]
Closed, will be handled by issue #839.

**Comment by jschreuder** [comment]
This isn't actually solvable, it's just not possible to have such a default for radio/checkboxes through these methods. The label won't know what the value is for the field and thus can't add that part. Within the 1.x codebase this will have to remain a limitation because any real solution to this will break backwards compatibility. In 2.0 this will be handled differently.
